scripts/toggle_logging.sh

```diff
@@ -4,15 +4,17 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 source "$CURRENT_DIR/variables.sh"
 source "$CURRENT_DIR/shared.sh"
+source "$CURRENT_DIR/capture_pane_helpers.sh"
 
-get_filename() {
-	local logging_path="$(get_tmux_option "$logging_path_option" "$default_logging_path")"
-	local logging_filename="$(get_tmux_option "$logging_filename_option" "$default_logging_filename")"
-	echo "${logging_path}/${logging_filename}"
-}
+# Functions defined in capture_pane_helpers are customized via below variables.
+default_path="$default_logging_path"
+default_name="$default_logging_filename"
+
+path_option="$logging_path_option"
+name_option="$logging_filename_option"
```

Comment:
I just reviewed how and why are these variables defined. I must say it was a lousy decision on my part to set this "global state" that is then used implicitly in functions defined in `capture_pane_helpers.sh`.

If you have a suggestion how to refactor this, that would be welcome. I think we just want to pass all arguments to functions explicitly. For example, instead of just `get_path` it would be ok to have `get_path "$logging_path_option" "$default_logging_path"`

Reply:
If you wanna refactor this, let's do it in a separate PR. No need to do all at once.

Reply:
Thanks for the advice. I will create separate PR for refactoring this